README.md

```diff
@@ -35,9 +35,8 @@ Windows
  (Note: Perl, NASM need to be included in machine's PATH variable)
 
 To build Intel® SGX SSL package in Windows OS:
-1. Download OpenSSL package into openssl_source/ directory. (tar.gz package, e.g. openssl-1.1.1q.tar.gz)
```

Comment:
Need to add steps to prepare the submodule code here as we remove the download step.

Reply:
Updated